README.md

```diff
@@ -61,6 +61,7 @@ See [docs.sysdig.com](https://docs.sysdig.com/en/on-premises-upgrades.html#UUID-
 
 ### Version 6.x
 
+- [6.5.1 Release, October 2023]()
```

Comment:
@radhikapc here you should fill in the link from the Sysdig release webpage when it is already up, thank you.